code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -2,6 +2,7 @@
 /datum/species/human/felinid
 	name = "Felinid"
 	id = "felinid"
+	say_mod = "meows"

```

Comment:
```suggestion
	say_mod = "rasps"
```

Reply:
Every felinid is a pack a day smoker.

Reply:
That gives me an idea for another pr actually